themes/rusted/templates/index.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="row">
+  <div class="col-md-12 text-center">
+    <h1 class="pitch">Handpicked Rust updates, <br /> delivered to your inbox.</h1>
+    <p class="subtext">Stay up to date with events, learning resources, and recent developments in Rust community.</p>
```

Comment:
It's a bit more complicated than that. If I add four more characters ("the "), it will fall to a second line, so we'll either need to reduce the font or maybe remove "recent ". Or, we can add some more text and make the font larger so that both lines are reasonably wide (and get rid of the `h1` line).
Maybe we can also throw in "... curated by humans ..." somewhere there to add to the length if needed. :-)
/cc @brson